.pipelines/templates/.builder-release-template.yaml

```diff
@@ -105,6 +105,9 @@ steps:
         -e SGX_INSTALL=${SGX_INSTALL} \
         -e ENABLE_CGROUPV2=${ENABLE_CGROUPV2} \
         -e IMAGE_VERSION=${IMAGE_VERSION} \
+        -e PRIVATE_PACKAGES_URL="${PRIVATE_PACKAGES_URL}" \
```

Comment:
nit: rename to `PRIVATE_PACKAGE_URLS`?

Reply:
Although i don't have a strong preference and private_packages_urls makes more sense to me, decided the the current private_packages_url to keep it consistent with current Windows variable naming for the same.